brave/browser-laptop
# Add missing location to "new bookmark" button on about:bookmarks

Add missing location to "new bookmark" button on about:bookmarks

- update heading on Add/Edit bookmark control to be aware of all add/edit scenarios
- location now included on new bookmark (from about:bookmarks)
- cleanup on aboutActions

Fixes https://github.com/brave/browser-laptop/issues/5031

Auditors: @jkup @cezaraugusto

Test Plan:
1. Launch Brave and go to about:bookmarks
2. Click the Star+ icon (on the top right area of screen, above bookmarks in column header)
3. Notice control now says "Create New Bookmark" and includes location :)
